.gitignore

```diff
@@ -7,7 +7,16 @@ local.properties
 .idea/
 *.iml
 
+# Eclipse
+.project
+.classpath
+.settings
+
+
 # Gradle
 
 .gradle/
 build/
+
+# Maven
```

Comment:
You mean the an empty line after the header is missing? I'd say that there is an empty line too much after all other section markers :)